plugins/services/src/js/components/modals/ServiceDestroyModal.js

```diff
@@ -122,7 +168,9 @@ class ServiceDestroyModal extends React.PureComponent {
     }
 
     return (
-      <h4 className="text-align-center text-danger flush-top">{errorMsg}</h4>
+      <Trans render="h4" className="text-align-center text-danger flush-bottom">
+        {errorMsg}
```

Comment:
this probably won't work. you can only use `Trans` like this for static text, for variables you need to mark them and then used the marked text as the `id` for the the `Trans` component. if we can't mark the text because it comes from the server or is built, then it can't be used inside a `Trans` component.

Reply:
I just checked and it works (changed language to Chinese and set "123" as translation).
![Screenshot from 2019-03-29 16-38-07](https://user-images.githubusercontent.com/40791275/55240213-0dd62280-5241-11e9-846e-0e87afd864fd.png)

Reply:
It doesn't break, but it also doesn't function as you would expect. using a `Trans` with just a variable inside extract's a message of `"{errorMsg}"`. That doesn't do anything and is confusing.

Reply:
Okay, I will remove the translation for now. It is out of scope anyway.